docs/sql-reference/sql-statements/data-manipulation/SHOW PARTITIONS.md

```diff
@@ -15,18 +19,69 @@ SHOW [TEMPORARY] PARTITIONS FROM [db_name.]table_name [WHERE] [ORDER BY] [LIMIT]
 > This syntax only supports StarRocks tables (`"ENGINE" = "OLAP"`). For Elasticsearch and Hive tables, use SHOW PROC '/dbs/db_id/table_id/partitions'.
 > Since v3.0, this operation requires the SELECT privilege on the specified table. For v2.5 and earlier versions, this operation requires the SELECT__PRIV privilege on the specified table.
 
+## Description of return fields
+
+```plaintext
++-------------+---------------+----------------+---------------------+--------------------+--------+--------------+-------+--------------------+---------+----------------+---------------+---------------------+--------------------------+----------+------------+----------+
+| PartitionId | PartitionName | VisibleVersion | VisibleVersionTime  | VisibleVersionHash | State  | PartitionKey | Range | DistributionKey    | Buckets | ReplicationNum | StorageMedium | CooldownTime        | LastConsistencyCheckTime | DataSize | IsInMemory | RowCount |
++-------------+---------------+----------------+---------------------+--------------------+--------+--------------+-------+--------------------+---------+----------------+---------------+---------------------+--------------------------+----------+------------+----------+
+```
+
+| **Field**                | **Description**                                              |
+| ------------------------ | ------------------------------------------------------------ |
+| PartitionId              | The partition ID.                                            |
+| PartitionName            | The name of the partition.                                   |
+| VisibleVersion           | The version number of the last successful load transaction. The version number increases by 1 with each successful load transaction. |
+| VisibleVersionTime       | The timestamp of the last successful load transaction.       |
+| VisibleVersionHash       | The hash value for the version number of the last successful load transaction. |
+| State                    | The state of the partition. Fixed value: `Normal`.           |
+| PartitionKey             | The partition key that consists of one or more partition columns. |
+| Range                    | The range of the partition, which is a right half-open interval. |
+| DistributionKey          | The bucket key of hash bucketing.                            |
+| Buckets                  | The number of buckets for the partition.                     |
+| ReplicationNum           | The number of replicas for a tablet in the partition.        |
+| StorageMedium            | Data storage device. HHD means hard disk drives. SSD means Solid-state drives. |
+| CooldownTime             | cooling time for data in the partition. After the cooling time,  data in this partition on SSDs is automatically migrated to HDDs by StarRocks. |
```

Comment:
After the cooling time,  data in this partition on SSDs is automatically migrated to HDDs by StarRocks. -> The data stored on SSDs will be automatically migrated to HDDs after their storage duration on SSDs exceeds the specified cooling duration.